Sources/BraintreeVisaCheckout/BTVisaCheckoutNonce.swift

```diff
@@ -0,0 +1,54 @@
+import Foundation
+
+#if canImport(BraintreeCore)
+import BraintreeCore
+#endif
+
+// A `nonce` representing a Visa Checkout card.
```

Comment:
nit: looks like we use `///` for comments in the rest of the SDK. Can we update this class and properties to match that? Same with `UserData`